typescript/db/src/resolvers/workspace.ts

```diff
@@ -78,10 +84,58 @@ const workspaces = async (
   );
 };
 
+type CreateTutorialDatasetProps = Context & {
```

Comment:
The `Props` suffix is for React. With more generic code, we tend to use the `Options` prefix.

```suggestion
type CreateTutorialDatasetOptions = Context & {
```